cli.py

```diff
@@ -34,25 +36,51 @@ class CabinetWrapper:
     """
 
     def __init__(self):
-        current_path = os.getcwd()
         join = os.path.join
-        self.config_path = join(current_path, 'test.data', 'secrets')
-        self.default_vault_path = join(current_path, 'test.data', 'vaults')
-        self.load_credentials()
-
-    def load_credentials(self):
+        self.base_path = join(os.path.expanduser('~'), '.config', 'cabinet')
+        self.config_file = join(self.base_path, 'cli.ini')
+        self.secrets_path = join(self.base_path, 'secrets')
+        self.vault_path = join(self.base_path, 'vaults')
+        self.config = ConfigParser()
+        self.config.read(self.config_file)
+
+    def load_credentials(self, vault_name=None, account_id=None):
         """
-        TODO: Implement reading credentials from stdin.
-        TODO: Implement reading credentials from config file.
+        It loads the vault name and account id from the configuration, then
+        it overrides the configuration with the cli options (if entered).
+        Finally, it prompts for the account password and opens the vault.
+
+        Params:
+        :param vault_name: The name of the vault
+        :type: String
+        :param account_id:
+        :type: String
+
+        TODO: Move the print and die to utils
         """
-        self.account_id = 'my-name@my-company.com'
-        self.password = 'asdfasdf'
-        self.vault_name = 'test-vault'
+
+        if vault_name:
+            self.vault_name = vault_name
+        elif self.config['Credentials']['default_vault']:
```

Comment:
This will raise an exception if there's no config file and you didn't specify a vault as a parameter, see:

```
➜ python cli.py search
Traceback (most recent call last):
  File "cli.py", line 302, in <module>
    app.run()
  File "/home/ivan/.virtualenvs/cabinet/lib/python3.5/site-packages/cement/core/foundation.py", line 882, in run
    return_val = self.controller._dispatch()
  File "/home/ivan/.virtualenvs/cabinet/lib/python3.5/site-packages/cement/core/controller.py", line 471, in _dispatch
    return func()
  File "/home/ivan/.virtualenvs/cabinet/lib/python3.5/site-packages/cement/core/controller.py", line 477, in _dispatch
    return func()
  File "cli.py", line 284, in default
    if cab.load_credentials(vault_name, account_id):
  File "cli.py", line 64, in load_credentials
    elif self.config['Credentials']['default_vault']:
  File "/usr/lib/python3.5/configparser.py", line 956, in __getitem__
    raise KeyError(key)
KeyError: 'Credentials'
```

You should